include/irr/builtin/material_compiler/glsl/common.glsl

```diff
@@ -0,0 +1,1386 @@
+#ifndef _IRR_BUILTIN_MATERIAL_COMPILER_GLSL_COMMON_INCLUDED_
+#define _IRR_BUILTIN_MATERIAL_COMPILER_GLSL_COMMON_INCLUDED_
+
+#include <irr/builtin/material_compiler/glsl/common_declarations.glsl>
+
+#ifndef _IRR_USER_PROVIDED_MATERIAL_COMPILER_GLSL_BACKEND_FUNCTIONS_
+	#error "You need to define 'vec3 irr_glsl_MC_getCamPos()', 'instr_t irr_glsl_MC_fetchInstr(in uint)', 'bsdf_data_t irr_glsl_MC_fetchBSDFData(in uint)' functions above"
```

Comment:
instead of `irr_glsl_MC_getCamPos` we should require `irr_glsl_MC_getNormalizedWorldSpaceV()`

Reply:
still TODO

Reply:
still a todo